collect_app/src/main/java/org/odk/collect/android/database/helpers/InstancesDatabaseHelper.java

```diff
@@ -107,7 +111,8 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
     public void onDowngrade(SQLiteDatabase db, int oldVersion, int newVersion) {
         try {
             Timber.i("Downgrading database from version %d to %d", oldVersion, newVersion);
-            moveInstancesTableToVersion5(db);
+
+            downgradeToVersion5(db);
```

Comment:
The idea is for this migration to always work no matter what columns may be added or removed in the future. That's why there are the v7000 tests. This down migration should always end up with the correct columns for v5.

I had to rearrange this for introducing a v6 and you can pick up most of that change from https://github.com/opendatakit/collect/pull/3447/commits/945bc8fdab815687a83f4e2f7e2bc1f0aa19b666. Let's merge this first and then I'll rebase and fix conflicts over there.

Reply:
Ahhh right because it's called `moveInstancesTableToVersion5` i missed that it's a universal way for the future.